src/components/Sum.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Sum = ({ a, b }) => {
+  function getSum(x, y) {
+    return x + y;
+  }
+
+  const sum = getSum(a, b);
```

Comment:
For more complicated operations - this is okay.
Here it is unnecessary. Just use `a + b` in JSX